File: src/main/java/com/win/dfas/vo/request/FileReqVO.java
package com.win.dfas.vo.request;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.win.dfas.common.util.LongJsonDeserializer;
import com.win.dfas.common.util.LongJsonSerializer;
import com.win.dfas.common.vo.BaseReqVO;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import org.hibernate.validator.constraints.Length;

import javax.validation.constraints.NotNull;

/**
 * 包名称：com.win.dfas.vo.request
 * 类名称：FileReqVO
 * 类描述：FileReqVO
 * 创建人：@author wanglei
 * 创建时间：2019/6/11/9:38
 */
@ApiModel(value = "文件查询VO")
@Data
public class FileReqVO extends BaseReqVO {
    private String name;
    /**
     * md5
     */
    private String md5;

    private int filepathId;
}

File: src/main/frontend/src/page/file/vo/FileReqVO.ts
/****************************************************
 * 创建人：  @author wanglei
 * 创建时间: 2019-07-11 13:55:33
 * 项目名称：dfas-common-schedule
 * 文件名称: FileReqVO.ts
 * 文件描述: @Description: 文件查询 请求VO
 *
 * All rights Reserved, Designed By 投资交易团队
 * @Copyright:2016-2019
 *
 ********************************************************/
import { BaseReqVO } from "../../common/vo/BaseVO";

/**
 * <p>
 * 基础任务表 请求VO
 * </p>
 *
 * @author wanglei
 * @since 2019-07-11 13:55:33
 */

export default class FileReqVO extends BaseReqVO {
    /**
     * 文件名称
     */
    public name: string;
    /**
     * MD5
     */
    public md5: string;
    /**
     * 文件路径
     */
    public filepathId: number;
}
